concourse/pipeline.yml

```diff
@@ -141,3 +141,7 @@ jobs:
         params:
           URL: 'https://coronavirus-vulnerable-people.service.gov.uk/live-in-england'
           MESSAGE: "Checks that the application deployed to production is not serving HTTP error codes. If this fails, you should investigate immediately."
+      - task: run-smoke-tests
```

Comment:
Is this a duplicate for line 106?

Reply:
@bilbof already has a [PR](https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/pull/260) to resolve this in the vulnerable form.